include_core/omrcomp.h

```diff
@@ -609,7 +609,7 @@ typedef struct U_128 {
 #define OMR_ALIGNOF(x) alignof(x)
 #endif /* defined(_MSC_VER) && (1900 > _MSC_VER) */
 
-#if (__cplusplus >= 201103L)
+#if defined(__cplusplus) && (__cplusplus >= 201103L)
```

Comment:
The corresponding `#endif` also needs to be updated:
```
#endif /* defined(__cplusplus) && (__cplusplus >= 201103L) */
```

Reply:
Thank you, you are quite right. I believe I've now amended these other references as requested, and added the signature block to the commit message.